packages/main/src/TextArea.js

```diff
@@ -427,8 +428,15 @@ class TextArea extends UI5Element {
 		return this.getDomRef().querySelector("textarea");
 	}
 
-	_onkeydown() {
+	_onkeydown(event) {
 		this._keyDown = true;
+
+		if (isEscape(event)) {
+			const nativeTextArea = this.getInputDomRef();
+
+			this.value = this._lastValue;
```

Comment:
Should we fire the "input" event here?

Reply:
yes :D

Reply:
Just to ask, since I am not quite sure :D As a reference I checked the StepInput.js where similar logic is applied onkeydown, but no event is fired. Also I checked the same logic in ui5, there change and live change are fired. Maybe change event is more relevant? There is actually no input event happening, only value change.

Reply:
liveChange is fired in OpenUI5's text area and it makes sense to fire input here. I would consider the StepInput logic here as a bug.

Reply:
Done.